modules/Classroom/components/ExamCard/ExamCard.tsx

```diff
@@ -18,10 +23,35 @@ const ExamCard = ({ exam, classroomId }: TExamCardProps) => {
   const { claim } = useAppSelector(authenticatedUserSelector);
 
   const [editOpened, { open: openEdit, close: closeEdit }] = useDisclosure(false);
+  const [deleteOpened, { open: openDelete, close: closeDelete }] = useDisclosure(false);
```

Comment:
```suggestion
  const [deleteModalOpened, { open: openDeleteModal, close: closeDeleteModal }] = useDisclosure(false);
```

NIT - Best to be clear and precise with the names. Same applies for other similar vars throughout the file.